### PR TITLE
add annotation to sample; troubleshooting instructions

### PR DESCRIPTION
**Pull request by acpana**
Some small changes to the getting started guide that I'd have found useful going thru and a small fix potentially to the sample from the getting started guide.

### CODE REVIEW

**Review by jingyih — COMMENTED** [review]
/lgtm

Thank you for improving the doc!

File: config/samples/resources/artifactregistryrepository/artifactregistry_v1beta1_artifactregistryrepository.yaml [hunk 1]
```diff
@@ -18,6 +18,9 @@ metadata:
   name: artifactregistryrepository-sample
   labels:
     label-one: "value-one"
+  annotations:
```
Comment (justinsb) [inline config/samples/resources/artifactregistryrepository/artifactregistry_v1beta1_artifactregistryrepository.yaml:21]: Just FYI these samples do make it into our docs (and tests), so we have to be a little careful.  That said this is done in a bunch of other places so should be fine.  _Maybe_ we shouldn't be referencing the sample here, but I'm not sure I feel too strongly about that.

**Comment by justinsb** [comment]
Looks good but please fix that typo.

Also the test is failing because:
`ERROR: Resource docs must be regenerated. Please run 'make ready-pr' or 'make resource-docs' and update your PR.`

So we are checking that the generated docs (which do include the sample file you changed) are up-to-date - that's a good thing, I think!

**Pull request merged**